test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -131,14 +91,14 @@ public class ContentResultTest
                         new MediaTypeHeaderValue("text/foo"),
                         "κόσμε",
                         null,
-                        "text/foo; charset=utf-8",
+                        "text/foo",
```

Comment:
:ok: After offline conversations, I'm fine w/ the product code change leading to this test change. :ok: